docs/getting-started-guides/ubuntu/logging.md

```diff
@@ -18,3 +18,43 @@ The `juju debug-log` will show all of the consolidated logs of all the Juju agen
 See the [Juju documentation](https://jujucharms.com/docs/stable/troubleshooting-logs) for more information.
 
 
+## Managing log verbosity
+
+Log verbosity in Juju is set at the model level. You can adjust it at any time:  
+
+```
+juju add-model k8s-development --config logging-config='<root>=DEBUG;unit=DEBUG'
+```
+
+and later
+
+```
+juju config-model k8s-production --config logging-config='<root>=ERROR;unit=ERROR'
+```
+
+In addition, the jujud daemon is started in debug mode by default on all controllers. To remove that behavior edit ```/var/lib/juju/init/jujud-machine-0/exec-start.sh``` on the controller node and comment the ```--debug``` section. 
```

Comment:
Interesting. Can we xpost a bug to the juju tracker or do we have one already tracking this? It seems weird that we're asking users to remote into a unit and edit the startup file instead of making this a model config flag.  (non-blocking comment)